src/components/feeds/FeedList.tsx

```diff
@@ -3,23 +3,30 @@ import FeedItem from './FeedItem';
 import style from './feedList.module.scss';
 import BASE_URL from '../../utils/baseUrl';
 import { Article } from '../../types/article';
+import { useAppSelector } from '../../hooks/redux';
+import getFilteredTags from '../../redux/selectors/filterTagsSelector';
 
 const FeedList: FC = () => {
   const [articles, setArticles] = useState<Article[]>([]);
+  const { entities, tag } = useAppSelector(getFilteredTags);
+
+  const filteredArticles = entities.articles;
+
+  const fetchData = async () => {
+    try {
+      const response = await fetch(`${BASE_URL}/articles`);
+      const result = await response.json();
+      setArticles(result.articles);
+    } catch (e) {
+      const errorMessage = 'Something went wrong';
```

Comment:
For future - it's a not good way to handle errors - it would be better if you create an error state which will show an error toast to user about this error.